staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -271,6 +271,11 @@ func (r *Reflector) ListAndWatch(stopCh <-chan struct{}) error {
 			// We want to avoid situations of hanging watchers. Stop any wachers that do not
 			// receive any events within the timeout window.
 			TimeoutSeconds: &timeoutSeconds,
+			// Enable bookmark to keep the resourceVersion up to date. We do not
+			// assume bookmarks are returned at any specific interval, the server
+			// do not support bookmark will just ignore this field.
+			// Disabled in the alpha release.
```

Comment:
nit:
To reduce load on kube-apiserver on watch restarts, you may enable watch bookmarks.
Reflector doesn't assume bookmarks are returned at all (if the server do not support watch
bookmarks, it will ignore this field).
Disabled in Alpha release of watch bookmarks feature.